src/sonic-py-common/sonic_py_common/multi_asic.py

```diff
@@ -487,3 +492,18 @@ def get_asic_presence_list():
         # This is not multi-asic, all asics should be present.
         asics_list = list(range(0, get_num_asics()))
     return asics_list
+
+
+def is_front_panel_port(port: str, role=None):
+    """
+    @summary: This function will check if the interface is a front-panel port
+    @return:  Boolean
+    """
+    if port.startswith((backplane_prefix(), inband_prefix(), recirc_prefix())):
```

Comment:
Can't we just use role to differentiate? Why do we need to check prefix?

Reply:
To preserve backward compatibility. This call is used in other parts of the code and works for legacy which uses prefix and the new ones which use role